added area (=0) attribute to Segment2D class

KatsuragiCSL opened the pull request:
#### References to other Issues or PRs
Fixes #20460 


#### Brief description of what is fixed or changed
added area (=0) attribute to Segment2D class

<!-- BEGIN RELEASE NOTES -->
NO ENTRY
<!-- END RELEASE NOTES -->

KatsuragiCSL committed:
for fixing #20460

fixing issue https://github.com/sympy/sympy/issues/20460, by adding area (=0) attribute to Segment2D class

oscarbenjamin commented:
Looks like the tests have failed because of trailing whitespace:
```
AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/geometry/line.py, line 2369.
```

wsdookadr commented on sympy/geometry/line.py:
```diff
@@ -2365,7 +2366,10 @@ class Segment2D(LinearEntity2D, Segment):
 
     """
-    def __new__(cls, p1, p2, **kwargs):
+    
```
[The failing test](https://travis-ci.org/github/sympy/sympy/jobs/745076406#L325) is saying this training whitespace should be deleted.

KatsuragiCSL committed:
delete trailing whitespace

delete trailing whitespace as failing test in https://github.com/sympy/sympy/pull/20469

smichr commented:
-1 
I don't think this is a good idea and gave reasons on the issue page.

KatsuragiCSL committed:
Merge branch 'sympy:master' into master

smichr commented:
closing due to inactivity
